src/main/java/model/Service.java

```diff
@@ -0,0 +1,35 @@
+package model;
+
+
+import java.util.List;
+
+public class Service {
+
+    private ProductRepo productRepo = new ProductRepo();
```

Comment:
Wow! Ich bin tatsächlich überrascht, dass du die Repos hier initialisiert, statt sie außerhalb des Services zu erstellen und dann in den Konstruktor zu geben (das ist die häufigste Variante)

Da es hier allerdings funktioniert die Repos direkt zu initialisieren, sparst du dir etwas Arbeit beim Aufsetzen - bravo!
Ein möglicher Nachteil daran: Wenn du die Repositories direkt manipulieren möchtest, um Daten zum Testen aufzubereiten etc. wird das natürlich sehr schwierig in diesem Fall.

Kombination der beiden Möglichkeiten wäre es zwei Konstruktoren zu haben.
Einen leeren in dem die Repos direkt initialisiert wird und und einen indem man die zwei Repos von außen reingibt.